vueapp/src/components/Court.vue

```diff
@@ -1,29 +1,80 @@
 <template>
-  <div>
-    <!-- A court with a game on it -->
-    <Game
-      v-if="court && court.game"
-      :game="court.game"
-      :court="court"
-      class="courtcard"
-    ></Game>
-    <!-- A court with no game playing on it -->
-    <v-card v-else-if="court" class="pa-2 ma-2 court-card">
-      <v-card-title>{{ court.name }}</v-card-title>
-      <v-card-subtitle>Available</v-card-subtitle>
-    </v-card>
-    <!-- Empty space (no court) in the grid -->
-    <v-card v-else class="pa-2 ma-2 court-card"> </v-card>
-  </div>
+  <draggable
+    v-model="courts"
+    item-key="id"
+    v-bind="dragOptions"
+    @drop="drop"
+    @dragstart="drag"
+  >
+    <template #item="{ element: court }">
+      <div>
+        <!-- A court with a game on it -->
+        <Game
+          v-if="court && court.game"
+          :game="court.game"
+          :court="court"
+          class="courtcard"
+        ></Game>
+        <!-- A court with no game playing on it -->
+        <v-card v-else-if="court" class="pa-2 ma-2 court-card">
+          <v-card-title>{{ court.name }}</v-card-title>
+          <v-card-subtitle>Available</v-card-subtitle>
+        </v-card>
+        <!-- Empty space (no court) in the grid -->
+        <v-card v-else class="pa-2 ma-2 court-card"> </v-card>
+      </div>
+    </template>
+  </draggable>
 </template>
 
-<style>
+<style scoped>
```

Comment:
[This prevents CSS changes from applying to child components](https://vuejs.org/api/sfc-css-features.html), but there isn't any CSS in this file.